Makefile

```diff
@@ -16,14 +16,11 @@ TEST_IMAGE=$(REPO)-testimages:$(VERSION)
 GOOS=linux
 GOARCH=amd64
 
-.PHONY: all dep build check clean test
+.PHONY: all  build check clean test
```

Comment:
Please update development documentation under `doc/development.md` to reflect right way to pull dependencies after moving to go modules.

Reply:
Gopkg.toml is needed only by `dep` please remove that file.

Reply:
I have updated the document.

Reply:
I have updated the document.